src/components/VaccineProgressBar/VaccineProgressBar.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { ParentSize } from '@vx/responsive';
+import { v4 as uuidv4 } from 'uuid';
+import { ProgressBarContainer, StyledSvg } from './VaccineProgressBar.style';
+import { COLOR_MAP, VACCINATIONS_COLOR_MAP, vaccineColor } from 'common/colors';
+import { formatPercent } from 'common/utils';
+
+export interface ProgressBarProps {
+  // TODO(michael): Remove once we migrate the location page to the new version.
+  oldVersion?: boolean;
+  locationName: string;
+  vaccinationsInitiated: number;
+  vaccinationsCompleted: number;
+}
+
+function getOffsetPercentage(decimal: number) {
+  return formatPercent(decimal, 1);
+}
+
+const VaccineProgressBar: React.FC<ProgressBarProps & { width: number }> = ({
+  oldVersion = false,
+  vaccinationsInitiated,
+  vaccinationsCompleted,
+  locationName,
+  width,
+}) => {
+  const height = 16;
```

Comment:
nit: since we're adding a 1px gray border, want to bump this height up by 2px?

Reply:
So I _think_ `16` correctly matches the mocks...  so I'm gonna' keep this as-is unless I'm confused or you think it looks funky / whatever.

I do think it maybe looks a tiny bit squashed on the location page.  Maybe we should make height configurable and bump the location page to 18. 🤷

Reply:
ooo, yeah im just being nitpicky -- the therm content/the blue bars have a height of 16px in the mocks, but when we add a 1px border to the svg, the content/blue bars shrink to 14px (16px svg - 1px border on top - 1px border on bottom)

![Screen Shot 2021-06-21 at 5 32 11 PM](https://user-images.githubusercontent.com/44076375/122830637-9de76300-d2b6-11eb-881b-72cb4de1e9b9.png)

Reply:
the mocks have the gray border only around the white bar, but this implementation has it around the entire svg, incl the blue bars, so the blue bars end up a touch smaller. fwiw--getting the border around only the white part (and only on 3 sides) feels like a bit of a pain

also, we're talking a total of 2px, so feel free to ignore overall 😂

Reply:
Ohhhhh...  I see.  Yeah, I didn't notice that the grey border only went around the white part in the Mocks.  Nice attention to detail! 

I think to match that we'd have to re-do it as 3 divs instead of SVG, so we can omit the left border on the white part...?  Probably doable, but I think I'll just bump the height to 18 and then maybe subtract a pixel of margin above/below to compensate in the tooltip. 😁